### PR TITLE
[DOCFIX] Add doctor storage docs

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

**Comment by LuQQiu** [comment]
@ns1123 PTAL, thanks

File: docs/en/operation/Admin-CLI.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ The `doctor` command shows Alluxio errors and warnings.
 ```bash
 # shows server-side configuration errors and warnings
 ./bin/alluxio fsadmin doctor configuration
+# shows worker lost storage warnings
```
Comment (ns1123) [inline docs/en/operation/Admin-CLI.md]: I would reword to '# shows worker storage health errors and warnings'

Comment (LuQQiu) [reply docs/en/operation/Admin-CLI.md]: Updated, thanks!

**Comment by LuQQiu** [comment]
@calvinjia  Please help merge this PR, thanks!

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.
